### PR TITLE
Nyt data

### PR DESCRIPTION
**Pull request by sollipse**
Depends on #11

### CODE REVIEW

**Comment by prendradjaja** [comment]
changed base branch for easier diffing

File: src/app/nyt-fetcher.service.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    .map((info: any) => {
+      let date = parseCDSDateString(info.date);
+      return {
+        ...info,
+        date,
+        daysSinceOutbreak: differenceInCalendarDays(date, outBreakStartDate),
+      };
+    });
+  return formattedDatesSinceOutbreak;
+}
```
Comment (prendradjaja) [inline src/app/nyt-fetcher.service.ts:40]: Would be nice to DRY this out (duplicated between here and CdsFetcherService)

File: src/app/nyt-fetcher.service.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+const convertNYTArrayToCoronaDatapoint = ([
```
Comment (prendradjaja) [inline src/app/nyt-fetcher.service.ts:41]: Probably we should have some datapoint concept of our own that we convert both NYT and CDS to (as opposed to converting NYT to CDS)

File: src/app/nyt-fetcher.service.ts [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+import differenceInCalendarDays from 'date-fns/differenceInCalendarDays';
+import parse from 'date-fns/parse';
+import mapValues from 'lodash/mapValues';
+import { Foo } from './cds-fetcher.service';
+import { from, Observable } from 'rxjs';
+
+const NYT_URL = `https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-states.csv`;
+
+const parseCDSDateString = (dateString) =>
+  parse(dateString, 'yyyy-MM-dd', new Date(1984, 0, 1));
```
Comment (prendradjaja) [inline src/app/nyt-fetcher.service.ts:13]: Duplicated with CDS. Anyway let's rename to "parseIsoDate" or something

**Review by prendradjaja — CHANGES_REQUESTED** [review]
A few things. Might be a bit involved. We could leave to a later PR or I could do some of these things. In any case maybe we get #11 merged first though before working on this!

**Comment by prendradjaja** [comment]
@sollipse Anything to keep from this branch or shall we close this PR now (bc of the Lambda)?

**Comment by sollipse** [comment]
No need, this is redundant

**Pull request closed without merging**